### PR TITLE
Add Raycast Icon

### LINKED ISSUE #8977: Raycast
### Brand Name

Raycast

### Website

https://www.raycast.com/

### Popularity Metric

Raycast's Similarweb rank is: [125,433](https://www.similarweb.com/website/raycast.com/#overview)

### Official Resources for Icon and Color

The svg ressources can be found at [Raycast Press Kit](https://raycastapp.notion.site/Raycast-Press-Kit-ce1ccf8306b14ac8b8d47b3276bf34e0)

The brand color is `#FF6363`

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #8977

**Similarweb rank:**
Raycast's Similarweb rank is: [125,433](https://www.similarweb.com/website/raycast.com/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This change resolves issue #8977 by adding Raycast logo in raycast.svg file and by updating simple-icons.json file with icon's data

### Preview
raycast

### CODE REVIEW

**Comment by RiteshK-611** [comment]
> ping @thomaspaulmann @itsnwa to double check the brand guidelines

Hey @fmerian I think its no required as I took svg from their official asset https://raycastapp.notion.site/Raycast-Press-Kit-ce1ccf8306b14ac8b8d47b3276bf34e0

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9585,6 +9585,12 @@
             "hex": "028CF0",
             "source": "https://github.com/ray-project/ray/blob/6522935291caa120e83697c6c9b3a450617c9283/dashboard/client/src/logo.svg"
         },
+        {
+            "title": "Raycast",
+            "hex": "FF6363",
+            "source": "https://raycastapp.notion.site/Raycast-Press-Kit-ce1ccf8306b14ac8b8d47b3276bf34e0",
+            "guidelines": "https://raycastapp.notion.site/Raycast-Press-Kit-ce1ccf8306b14ac8b8d47b3276bf34e0"
```
Comment (adamrusted) [inline _data/simple-icons.json:9592]:
```suggestion
            "source": "https://www.raycast.com",
            "guidelines": "https://www.raycast.com/press"
```

File: icons/raycast.svg
Comment (adamrusted) [inline icons/raycast.svg:1]: There's a couple of [tiny differences](https://github.com/simple-icons/simple-icons/assets/1830867/7748ad2e-70e1-4690-9f3a-99650c2f61fe) between your icon and the one found on the website homepage. If you don't fancy re-doing this with the website header SVG, I'm happy to let this one through - else you'll be going up to 5 decimals of precision! Could you update the source and guidelines link though?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/raycast.svg
Comment (adamrusted) [inline icons/raycast.svg:1]: Im still seeing [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/4ff4edc8-33ba-4d11-b03b-aadcaf71e080) throughout this icon. @LitoMore - would you mind pitching in on this one?

**Review by PeterShaggyNoble — DISMISSED** [review]

**Review by adamrusted — APPROVED** [review]

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11122,6 +11122,12 @@
...
```
Comment (adamrusted) [inline _data/simple-icons.json:11129]:
```suggestion
            "source": "https://www.raycast.com/press",
            "guidelines": "https://www.raycast.com/press"
```

**Pull request merged**